docs/docs/fan-unit/control/uart.mdx

```diff
@@ -0,0 +1,86 @@
+
+import Admonition from '@theme/Admonition';
+
+# Smart
+
+The Smart Fan unit utilizes the [RP2040](https://www.raspberrypi.com/products/rp2040/specifications/) the Raspberry Pi microcontroller featured in thr [Raspberry Pi Pico 1](https://www.raspberrypi.com/documentation/microcontrollers/pico-series.html). Using the RP2040's two UART connections a link can be established out of to the to blades using the fan connector
+
+## Pins
+
+The A port on the Fan unit uses `UART0` and the B port uses `UART1` on the RP2040. On the CM4 connects using `UART5`. both blade connectors can be used to power the Fan unit with +5v.
+
+<details>
+  <summary>Pin Out Guides</summary>
+  <h3>Smart Fan Unit</h3>
+  <img src='/img/fan-unit/smart-gpio.svg' width={'70%'} alt="Smart Fan Unit blade connecter pin out"/>
+  <h3>Compute blade</h3>
+  <img src='/img/hardware/fan-port.webp' width={'70%'} alt="Blade Fan unit connector"/>
+  
+</details>
+
+
+## Update firmware
+
+To start using the the Smart Fan Unit it first should be flashed with the latest firmware of your choice. This could be custom firmware or a preexisting framework.
```

Comment:
```suggestion
To start using the Smart Fan Unit, it should first be flashed with the latest firmware of your choice. This could be any custom firmware, or a pre-existing framework.
```